mlir/lib/Dialect/SparseTensor/IR/SparseTensorDialect.cpp

```diff
@@ -374,7 +374,7 @@ Level SparseTensorEncodingAttr::getLvlRank() const {
 
 LevelType SparseTensorEncodingAttr::getLvlType(Level l) const {
   if (!getImpl())
-    return LevelFormat::Dense;
+    return LevelFormat::Batch;
```

Comment:
is this right now? I guess this is better?

Reply:
I think `Batch` make more sense than `Dense` for non-annotated tensor here, because it would require keeping all non-linearized coordinates.